algorithms/integration/kapton_2019_correction.py

```diff
@@ -350,24 +338,14 @@ def _check_int_edge_pts(int_edge_pts):
         # Now make sure the edges and the kapton lines are on the right side (i.e not swapped).
         # Let's look at edge_idx[0:2] i,e the first edge of detector parallel to the kapton
         pt1 = edge_pts[edge_idx[0]]
-        pt2 = edge_pts[edge_idx[1]]
         # Now find the distance between each of these points and the kapton lines.
         d1_kapton_1 = self.distance_of_point_from_line(
             pt1, int_edge_pts[0], int_edge_pts[1]
         )
         d1_kapton_2 = self.distance_of_point_from_line(
             pt1, int_edge_pts[2], int_edge_pts[3]
         )
-        d2_kapton_1 = self.distance_of_point_from_line(
-            pt2, int_edge_pts[0], int_edge_pts[1]
-        )
-        d2_kapton_2 = self.distance_of_point_from_line(
-            pt2, int_edge_pts[2], int_edge_pts[3]
-        )
         if d1_kapton_1 < d1_kapton_2:  # closer to max than edge
-            assert (
```

Comment:
This assertion breaks the ability to calculate absorption edge/max for higher angles of Kapton tape. 

It seems to me that this now deleted logic is enforcing a relative orientation of the kapton tape absortion max and edge with detector edges that isn't necessarily always true. That said, I'm not sure what is lost by deleting this logic. I did find, however, that the deletion didn't result in any weird behavior at lower angles (negative or positive).